extensions/amp-mega-menu/validator-amp-mega-menu.protoascii

```diff
@@ -56,28 +63,46 @@ tags: {  # <amp-mega-menu> <nav>
     tag_spec_name: "AMP-MEGA-MENU NAV > UL/OL"
   }
 }
-tags: {  # <amp-mega-menu> <template> <nav>
+tags: {
   html_format: AMP
-  tag_name: "NAV"
-  spec_name: "AMP-MEGA-MENU TEMPLATE > NAV"
-  satisfies: "amp-mega-menu nav"
-  mandatory_parent: "TEMPLATE"
-  mandatory_ancestor: "AMP-MEGA-MENU"
-  disallowed_ancestor: "NAV"
-  siblings_disallowed: true
+  tag_name: "$REFERENCE_POINT"
+  spec_name: "AMP-MEGA-MENU > AMP-LIST"
+  # Include mandatory amp-list attrs so that <amp-mega-menu> > <nav>
```

Comment:
Neat hack. We're lucky that the two can be differentiated by attributes. It seems a bit fragile given it's dependent on the mandatory_anyof in validator-amp-list.protoascii (or else `<amp-list/>` would match this ref point), but I can't think of a better solution off the top of my head.

Reply:
Indeed; it'd be nice if we could distinguish reference points by tag name, but not sure if there's significant engineering cost associated with that.